### DISCUSSION TITLE
Main differences between self hosted and hosted version?

### DISCUSSION THREAD

**Discussion by maximilian-schwarz** [nhost/nhost#1521] [Q&A]
Whats the main difference between self hosted and hosted version?

Is it possible to host multiple projects in self hosted?

Also there are serverless functions available?

**Comment by nokia6290** [comment]
ping

**Reply by dbarrosop** [reply]
The self-hosted version will have the features you add to it, we provide docker images for hasura-auth and hasura-storage, the rest of the features will depend on you and your hosting provider. You can also host as many copies/projects as you want, how to do it will depend on your hosting provider.
